api/src/trace/SugaredTracer.ts

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type {Context, Span, Tracer} from '../';
+import * as api from '../';
+import {SpanStatusCode} from '../';
+import {SugaredSpanOptions} from './SugaredOptions';
+
+const defaultOnException = (e: Error, span: Span) => {
+  span.recordException(e);
+  span.setStatus({
+    code: SpanStatusCode.ERROR
+  });
+};
+
+/**
+ * return a new SugaredTracer created from the supplied one
+ * @param tracer
+ */
+export function wrapTracer(tracer: Tracer): SugaredTracer {
```

Comment:
Note that APIs need to be explicitly exported in https://github.com/open-telemetry/opentelemetry-js/blob/main/api/src/index.ts#L89.